app.js

```diff
@@ -0,0 +1,69 @@
+const { response } = require("express");
+const express = require("express");
+const { download } = require("express/lib/response");
+
+
+const pokemon = require("./models/pokemon.js")
+console.log(pokemon[1]);
+
+
+const app = express();
+
+app.get("/:verb/:noun/:adjective", (req, response) => {
+    const { verb, noun, adjective } = req.params;
+    response.send(
+        `Congratulations on starting a new project called ${verb}-${noun}-${adjective}`
+    );
+});
+
+
+app.get("/bugs/:numberOfBugs", (req, res) => {
+    const {numberOfBugs} = req.params;
+    if (numberOfBugs < 200) {
+        res.send(
+            `${numberOfBugs} litte bugs in the code` + `<a href = '/bugs/101 ${Number(numberOfBugs) + 2}'> Pull one down, patch it around</a>`
+        )
+    } else if (numberOfBugs < 200) {
+        res.send(
+            `${numberOfBugs} litte bugs in the code` 
+        );
+    }
+    else response.send(`Too many bugs!! Start over!`);
+})
```

Comment:
I see two issues in this block of code, one of them is keeping your tests from passing:
1) There's a typo. I see the same typo in two places
2) your conditions on lines 22 and 26 are identical. your else if statement is never going to do anything because it can only run if _the if statement that precedes it is **false**_ **and** _the else if statement is **true**_. This can never be the case if they're using the exact same condition.